deploy/template.yaml

```diff
@@ -1220,6 +1271,46 @@ Resources:
                   - "dynamodb.amazonaws.com"
                   - "ecs-tasks.amazonaws.com"
 
+  S3KmsKey:
```

Comment:
I think you may need to add SPOT to be able to use this KMS key so that it's able to encrypt and decrypt contents of the S3 bucket.

Reply:
Access policy for Spot on Kms key has been added.